code/API_definitions/qod-api.yaml

```diff
@@ -222,7 +222,46 @@ paths:
         - oAuth2ClientCredentials: []
         - threeLegged:
             - "qod-sessions-write"
-
+    get:
+      tags:
+        - QoS sessions
+      summary: Get all session information
+      description: Get information about all active sessions that were created by the client
+      operationId: getSessions
+      parameters:
+        - name: perPage
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/PerPage"
+        - name: page
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/Page"
+      responses:
+        "200":
+          description: Contains information about active sessions
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SessionInfoList"
```

Comment:
Yeah, I've seen it but I wasn't sure if this incomplete list of resources means that some resources are not allowed to be returned or just because of pagination. So now I understand that every time when user specifies the perPage value less than all active sessions count, then the response code should be 206 instead of 200 ?

Reply:
In the guideline there is also specified that when pagination is supported then the query parameter "seek" must be accepted. For now I skipped this parameter thinking that in our case it won't be very necessary. But maybe it will be better to also introduce it since it is required in the guidelines?

Reply:
About 206.  Yes I also think that should be returned if not the whole list of sessions is returned. 
About seek, I think the quideline is not completely clear, I would omit it now and let the community decide.

Reply:
Okay